compiler/cli/src/org/jetbrains/kotlin/cli/jvm/compiler/compat.kt

```diff
@@ -20,7 +21,10 @@ object IdeaStandaloneExecutionSetup {
         System.getProperties().setProperty("ide.hide.excluded.files", "false")
         System.getProperties().setProperty("ast.loading.filter", "false")
         System.getProperties().setProperty("idea.ignore.disabled.plugins", "true")
-        System.getProperties().setProperty("idea.home.path", System.getProperty("java.io.tmpdir"))
```

Comment:
Why did you drop `idea.home.path`? Was it simply a "safety measure" in order to prevent other unexpected reads from `/tmp` directory? Or do you have another reason to drop it? It looks like `idea.home.path` drop isn't necessary to fix the issue with the steps [you described](https://github.com/JetBrains/kotlin/pull/4763#issuecomment-1066710153)

Reply:
@goodwinnk do you remember why [did you add this line of code](https://github.com/JetBrains/kotlin/commit/87a520fc3018eaee92ad46a03aa5d07d9db4a37a) in the first place? From your commit message it's not clear what are steps to reproduce

Reply:
I'm happy to put it back. As far as I can tell, it will not be read now that I set the other one. However, I might be missing something, so maybe it is safer to leave it there. Let's see if @goodwinnk remembers why it was added and we can decide whether to put it back in.

Reply:
Also, I initially removed it to see if I could get the intellij code to hit the fallback path. Unfortunately, the intellij code thinks it is running in the IDE when exercised from kotlinc. That would be another way to fix this: remove just the idea.home.path system property, but then we would have to change something in intellij as well to somehow communicate that we are not running in the IDE. I couldn't immediately find a way to communicate that, and then I realized that if I just set the build number in a property to the fallback, I don't have to. I'm very open to suggestions if you see a better way of doing this.